addons/mail/static/src/core/web/activity_menu.js

```diff
@@ -82,6 +82,13 @@ export class ActivityMenu extends Component {
                 });
             return;
         }
+
+        if (filter === "all") {
+            context["search_default_activities_overdue"] = 1;
+            context["search_default_activities_today"] = 1;
+        } else {
+            context["search_default_activities_" + filter] = 1;
```

Comment:
Hello, quick comment as I see the PR.

If the list can be hard-coded it would be better, as building technical strings dynamically is a nightmare for maintenance (can't find them back in code with search, risk of passing unexpected values, etc).

Reply:
+1

Reply:
Thank you for reviewing the PR. I have made the suggested changes.